docs/components.md

```diff
@@ -21,28 +21,44 @@ KubeVirt consists of a set of services:
 
 ## Example flow: Create and Delete a VM
 
-    Client         Virt API    VM TPR  Virt Controller  k8s         VM Handler
-    -------------- ----------- ------- ---------------- ----------- ----------
-
-                   listen <----------- WATCH /vms
-                   listen <---------------------------------------- WATCH /vms
-                                          |                            |
-    POST /vms ---> validate               |                            |
-                   create ---> VM ---> observe                         |
-                     |          |         v                            |
-                     |          |      POST /pods ----> validate       |
-                     |          |         |             create         |
-                     |          |         |             schedPod -> observe
-                     |          |         |                            v
-                     | <------------------------------------------> GET /vms
-                     |          |         |                         defineVM
-                     |          |         |                         watchVM
-                     |          |         |                            |
-    DELETE /vms -> validate     |         |                            |
-                   delete ----> * --------------------------------> observe
-                     |                    |                         shutdownVM
-                     |                    |                            |
-                     :                    :                            :
+```
+Client         Virt API    VM TPR  Virt Controller  k8s         VM Handler
+-------------- ----------- ------- ---------------- ----------- ----------
+
+               listen <----------- WATCH /vms
+               listen <---------------------------------------- WATCH /vms
+                                      |                            |
+POST /vms ---> validate               |                            |
+               create ---> VM ---> observe                         |
+                 |          |         v                            |
+                 |          |      POST /pods ----> validate       |
+                 |          |         |             create         |
+                 |          |         |             schedPod -> observe
+                 |          |         |                            v
+                 | <------------------------------------------> GET /vms
```

Comment:
if you have a virt-handler running on node 'master', this virt-handler watches all VMs with a label `kubevirt.io/nodeName=master`. So if a VM gets such a label, or a VM with such a label changes or gets deleted, `virt-handler` get's notified.